Main.java

```diff
@@ -0,0 +1,12 @@
+import java.util.Scanner;
+    public class Main {
```

Comment:
Please change to more clear name which will describe what class do

Reply:
Class name is changed to PersonalData from Main